index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const webidlWrapper = require('./webidl2js-wrapper.js');
+const webidlConversions = require('webidl-conversions');
+
+const sharedGlobalObject = {};
+webidlWrapper.install(sharedGlobalObject);
+
+const origCSSStyleDeclaration = sharedGlobalObject.CSSStyleDeclaration;
+
+/**
+ * @constructor
+ * @param {((cssText: string) => void) | null} [onChangeCallback]
+ * The callback that is invoked whenever a property changes.
+ */
+function CSSStyleDeclaration(onChangeCallback = null) {
+  if (new.target === undefined) {
+    throw new TypeError('Illegal invocation');
+  }
+
+  if (onChangeCallback !== null) {
+    onChangeCallback = webidlConversions.Function(onChangeCallback, {
+      context: 'Failed to constructor CSSStyleDeclaration: parameter 1',
+    });
+  }
```

Comment:
Please don't use Web IDL conversions if it's not actually a Web IDL method. If/when we implement IDL callback functions in webidl2js properly this function might disappear, as [`Function`](https://heycam.github.io/webidl/#Function) is a typedef in Web IDL.